### PR TITLE
docs: update

### PR DESCRIPTION
**Pull request by andrewbanchich**
## Proposed changes

I went through the first couple pages of docs and fixed errors (wrong port numbers, hostnames, etc.), as well as spelling, punctuation, formatting, etc.

One thing I wanted to mention is referring to it as `ORY Kratos` vs `Kratos`. I felt that calling it `ORY Kratos` through the docs felt monotonous, and users already know that the company's name is ORY from the website, logo, Docker/NPM namespace, etc. so I opted to leave it as `ORY Kratos` in only a couple places and then refer to it as `Kratos` everywhere else.

Let me know if you aren't in agreement with that and I can change it back :smile:

## Checklist

- [X] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [X] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: docs/docs/quickstart.mdx [hunk 1]
```diff
@@ -82,22 +72,31 @@ const needsLogin = (req, res, next) => {
       res.redirect('/login');
     });
 };
+
+// You can use `needsLogin` as a middleware for Express: 
+// app.get("/dashboard", needsLogin, dashboard)
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:77]: I think the original example is clearer ("where does `app` come from?")

File: docs/docs/quickstart.mdx [hunk 2]
```diff
@@ -82,22 +72,31 @@ const needsLogin = (req, res, next) => {
...
-made, we forward (like a proxy) the request to
-`https://public.ory-kratos/self-service/browser/flows/login` and pipe the
-response back to the initial HTTP Request:
+Because our SecureApp and Kratos need to share cookies, in order for
+anti-CSRF tokens and login sessions to work, we will set up a path which forwards
+requests to Kratos' Public API so that both SecureApp and Kratos have the same hostname.
+
+If an HTTP request is made to
+
+`https://my-secureapp/.ory/kratos/public/self-service/browser/flows/login`
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:93]: Make this a code block with three back ticks :)

File: docs/docs/quickstart.mdx [hunk 3]
```diff
@@ -82,22 +72,31 @@ const needsLogin = (req, res, next) => {
...
+
+we forward (like a proxy) the request to
+
+`https://public.ory-kratos/self-service/browser/flows/login`
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:97]: Make this a code block with three back ticks :)

File: docs/docs/quickstart.mdx [hunk 4]
```diff
@@ -108,7 +107,7 @@ const pathPrefix = '/.ory/kratos/public';
 app.use(pathPrefix + '/', (req, res) => {
   const url = 'https://public.ory-kratos' + req.url.replace(pathPrefix, '');
 
-  // Uses the request library to forward the request to ORY Kratos
+  // Uses the `request` library to forward the request to Kratos
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:110]: We use `ORY` as a prefix consistently across all docs, product pages, manuals - this shouldn't be the odd one out so please revert it :)

The prefix `ORY` is important because there are other projects called e.g. Hydra and other things (e.g. Game of Thrones) for Oathkeeper.

File: docs/docs/quickstart.mdx [hunk 5]
```diff
@@ -117,52 +116,25 @@ app.use(pathPrefix + '/', (req, res) => {
...
-cd kratos
-git checkout v0.3.0-alpha.1
-make quickstart
-
-# or if you don't have make installed:
 docker pull oryd/kratos:latest-sqlite
-docker pull oryd/kratos-selfservice-ui-node:latest
+docker pull oryd/kratos-selfservice-ui-node
 docker-compose -f quickstart.yml -f quickstart-standalone.yml up --build --force-recreate
 ```
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:139]: Please revert these changes, otherwise the chat will be flooded again with people complaining about Docker not working.

File: docs/docs/quickstart.mdx [hunk 6]
```diff
@@ -171,203 +143,260 @@ healthy system you're ready to roll! A healthy system will show something along
...
+2. [SecureApp](http://github.com/ory/kratos-selfservice-ui-node)
+- Public (port 4455) - an example application written in NodeJS
+   that implements the login, registration, logout, dashboard, and other UIs.
+3. [MailHog](https://github.com/mailhog/MailHog)
+- Public (port 4436) - a development SMTP server which Kratos will use to send emails.
+
+To better understand the application architecture, let's take a look at the network
 configuration. This assumes that you have at least some understanding of how
-Docker (Compose) Networks work:
+Docker networks work:
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:186]:
```suggestion
Docker Networks work:
```

File: docs/docs/quickstart.mdx [hunk 7]
```diff
@@ -171,203 +143,260 @@ healthy system you're ready to roll! A healthy system will show something along
...
+hostname.
 
 ## Perform registration, logout, login
 
-Enough theory, it's time to get this thing going! Let's start by trying to open
-the dashboard - **go to
-[127.0.0.1:4455/dashboard](http://127.0.0.1:4455/dashboard)**. You will probably
-notice that you're ending up at the login endpoint:
+Enough theory! Let's start by trying to open
+the dashboard: go to [127.0.0.1:4455/dashboard](http://127.0.0.1:4455/dashboard).
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:196]:
```suggestion
the dashboard: go to **[127.0.0.1:4455/dashboard](http://127.0.0.1:4455/dashboard)**.
```

File: docs/docs/quickstart.mdx [hunk 8]
```diff
@@ -171,203 +143,260 @@ healthy system you're ready to roll! A healthy system will show something along
...
+
+You should notice that you're ending up at the login endpoint instead of the dashboard:
 
-<img
-  alt="Login screen of your secured app"
-  src={useBaseUrl('img/docs/secureapp-login.png')}
-/>
+<p>
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:200]: Markdown should wrap this automatically in a paragraph, no?

File: docs/docs/quickstart.mdx [hunk 9]
```diff
@@ -171,203 +143,260 @@ healthy system you're ready to roll! A healthy system will show something along
...
+an HTTP request to `http://kratos:4434/self-service/browser/flows/requests/login?request=<request-id>`. Notice the URI is `kratos:4434` because SecureApp
+is making a server-side HTTP request via Docker's private network to Kratos' Admin API.
+6. Kratos responded with data which SecureApp used to render the HTML login form.
+
+Because we exposed Kratos' Admin API on port `4434` of our host, we can use `curl` to see what Kratos responds with.
+Try visiting `http://127.0.0.1:4455/auth/login`, copying the generated `request-id`, and making the request yourself:
 
 ```shell script
-$ curl http://127.0.0.1:4434/self-service/browser/flows/requests/login?request=<request-id>
+~ > curl "http://127.0.0.1:4434/self-service/browser/flows/requests/login?request=<request-id>" | jq
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:235]: `$` is the standard way of indicating a shell across many open source docs:

```suggestion
$ curl "http://127.0.0.1:4434/self-service/browser/flows/requests/login?request=<request-id>" | jq
```

File: docs/docs/quickstart.mdx [hunk 10]
```diff
@@ -376,43 +405,49 @@ You can retrieve the email however by opening the MailSlurper UI at
...
+To understand what each of those configuration files does, consult the other chapters of
 this documentation.
 
 :::note
 
-To get a minimal version of ORY Kratos running, you need to set configuration
-items `identity.traits.default_schema_url` and `dsn`. You should also configure
-`urls.*_ui` because your users will end up at fallbacks otherwise.
+To get a minimal version of Kratos running, you need to set configuration
+values for [`identity.traits.default_schema_url`](https://github.com/ory/kratos/blob/v0.3.0-alpha.1/contrib/quickstart/kratos/email-password/.kratos.yml#L67) and [`DSN`](https://github.com/ory/kratos/blob/v0.3.0-alpha.1/quickstart.yml#L42). You should also configure
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:443]: If we fix the tag here, we need to update it in our release pipeline so that it stays up to date with the newest version.

**Comment by aeneasr** [comment]
One last thing - this PR removes all the images. I think that's an accident, right? Could you please revert the static directory to its old state?

File: docs/docs/quickstart.mdx (hunk 5 above)
Comment (andrewbanchich) [reply docs/docs/quickstart.mdx:139]: Ah, I was under the impression if you don't specify a tag that it was the same thing as specifying `latest`.

File: docs/docs/quickstart.mdx (hunk 6 above)
Comment (andrewbanchich) [reply docs/docs/quickstart.mdx:186]: I don't think Docker Network is a proper name, is it? Looking at [Docker's usage](https://docs.docker.com/network/), they refer to them as `Docker networks` because it's just a plain old network run by Docker, and not a brand name of some sort.

File: docs/docs/quickstart.mdx (hunk 8 above)
Comment (andrewbanchich) [reply docs/docs/quickstart.mdx:200]: It wasn't spacing things properly for me. The previous screenshots had some transparent padding in the PNGs themselves, but the ones I updated them to don't.

File: docs/docs/quickstart.mdx (hunk 10 above)
Comment (andrewbanchich) [reply docs/docs/quickstart.mdx:443]: True, but I was thinking if we specified a line number against `master` that the line might change in the future to be something other than what it's meant to refer to.

**Comment by andrewbanchich** [comment]
> One last thing - this PR removes all the images. I think that's an accident, right? Could you please revert the static directory to its old state?

Does it? The diffs show the size has decreased but that's because they were modified, not removed. I can view the images from the diff.

I replaced them with newer screenshots since I thought some of the paths were out of date, and also zoomed in on some to make them more readable.

File: docs/docs/quickstart.mdx (hunk 4 above)
Comment (andrewbanchich) [reply docs/docs/quickstart.mdx:110]: Okay, sounds good. I'll revert it.

However, just as a suggestion I would consider thinking about this in the future since it sounds like it's just SEO related and I'm guessing searching for "ORY Kratos" or "Kratos identity management" would probably bring up ORY Kratos and not God of War, even without using "ORY" everywhere "Kratos" is used.

File: docs/docs/quickstart.mdx (hunk 1 above)
Comment (andrewbanchich) [reply docs/docs/quickstart.mdx:77]: I thought about that, too, but I think the spirit of this comment is just to show "it can be used as middleware later" and not necessarily to show people how to import modules from Express. Since we're already assuming basic knowledge of JavaScript / Express, I thought it would be safe to remove boilerplate code just to highlight the API usage itself.

If you disagree, I can revert this (not a huge deal in my opinion).

File: docs/docs/quickstart.mdx (hunk 7 above)
Comment (andrewbanchich) [reply docs/docs/quickstart.mdx:196]: I removed the emphasis on this because the other links to `127.0.0.1` are not bold and wanted to remain consistent in style. Do you want all of these links bold?

File: docs/docs/install.md [hunk 11]
```diff
@@ -52,28 +52,28 @@ You can install ORY Kratos using [scoop](https://scoop.sh) on Windows:
 > kratos help
 ```
 
-## Download Binaries
+## Download binaries
```
Comment (aeneasr) [inline docs/docs/install.md:55]: We're using the New York Times headline capitalization as a rule of thumb everywhere. Check out: https://titlecaseconverter.com

For consistency, we should probably stay with that and fix it where we don't use it. Might also be good to document this.

File: docs/docs/install.md [hunk 12]
```diff
@@ -52,28 +52,28 @@ You can install ORY Kratos using [scoop](https://scoop.sh) on Windows:
...
 ```shell
 $ kratos help
 ```
 
-## Building from Source
+## Building from source
 
 If you wish to compile ORY Kratos yourself, you need to install and set up
-[Go 1.12+](https://golang.org/) and add `$GOPATH/bin` to your `$PATH`.
+[Go 1.12+](https://golang.org/) and add `$GOPATH/bin` to your path.
```
Comment (aeneasr) [inline docs/docs/install.md:72]: It's not clear what `path` refers to. `$PATH` is the correct thing to say here.

```suggestion
[Go 1.12+](https://golang.org/) and add `$GOPATH/bin` to your `$PATH`.
```

File: docs/docs/quickstart.mdx (hunk 6 above)
Comment (aeneasr) [reply docs/docs/quickstart.mdx:190]: Yup, my bad!

File: docs/docs/quickstart.mdx (hunk 8 above)
Comment (aeneasr) [reply docs/docs/quickstart.mdx:204]: Interesting, then we'll use the paragraph!

File: docs/docs/quickstart.mdx (hunk 10 above)
Comment (aeneasr) [reply docs/docs/quickstart.mdx:447]: Yes, that's true. So to keep it as part of our update pipeline I think that all you need to do is add quickstart.mdx here: https://github.com/ory/kratos/blob/master/docs/config.js#L20

File: docs/docs/quickstart.mdx (hunk 1 above)
Comment (aeneasr) [reply docs/docs/quickstart.mdx:77]: I'd prefer to have this reverted. Explicit documentation is better than implicit. No room for interpretation of things but clear, concise technical instructions. You can comment `import from express` with ("or any other web framework")

File: docs/docs/quickstart.mdx [hunk 13]
```diff
@@ -127,40 +130,13 @@ SMTP server the demo uses to show how e.g. email verification works.
...
-
-:::
-
-:::tip
-
-If you encounter build errors (e.g. network timeout), make sure that the network
-is running correctly and run `macke docker` again. If the problem persists, feel
-free to [open an issue](https://github.com/ory/kratos/issues/new/choose).
-
-:::
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:151]: Looks like my comment got lost somehow in the GitHub UI void :D

This is very important, we have tons of users that mess this up and come asking us for questions "why does docker say out of disk space????" on GitHub, chat, forum. Please revert this.

File: docs/docs/quickstart.mdx [hunk 14]
```diff
@@ -127,40 +130,13 @@ SMTP server the demo uses to show how e.g. email verification works.
...
 
 ```shell script
-git clone https://github.com/ory/kratos.git
-# or if you have git+ssh set up:
-#  git clone git@github.com:ory/kratos.git
-cd kratos
-git checkout v0.3.0-alpha.1
-make quickstart
-
-# or if you don't have make installed:
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:163]: Please revert this. It's very unclear what needs to be done without these instructions.

File: docs/docs/quickstart.mdx (hunk 13 above)
Comment (andrewbanchich) [reply docs/docs/quickstart.mdx:151]: Sounds good! I wasn't aware that was such a common issue 😄

File: docs/docs/quickstart.mdx (hunk 14 above)
Comment (andrewbanchich) [reply docs/docs/quickstart.mdx:163]: I'm a bit unclear on this part. Why provide instructions for `make` here at all? Docker's commands require only the necessary software (Docker and Docker Compose) so using `make` seemed a little tangential to just getting users up and running quickly, and I think if anything it obfuscates what is actually going on with Docker.

The plan Docker instructions underneath this, in my opinion, make it much clearer what's happening.

Comment (aeneasr) [reply docs/docs/quickstart.mdx:163]: Ok, that's a good point! Let's remove the `make quickstart` then. But we still need to check out the GitHub repository, otherwise all the config files will be missing!

File: docs/docs/quickstart.mdx (hunk 7 above)
Comment (aeneasr) [reply docs/docs/quickstart.mdx:222]: SGTM

**Comment by aeneasr** [comment]
Thank you for sticking with this lengthy review! Really appreciate your effort :)

File: docs/docs/quickstart.mdx (hunk 14 above)
Comment (andrewbanchich) [reply docs/docs/quickstart.mdx:163]: Ah, that makes sense! Fixed.

**Comment by andrewbanchich** [comment]
You're welcome! Thanks for sticking with it as well.

**Comment by aeneasr** [comment]
Thank you! Merged! :) I think we should document a style guide for the docs somehow. I've never actually specified the things mentioned here (e.g. NYT style headlines) and the use is also not consistent across the repository. I was thinking we could put that into the core docs maybe? What's your take on this?

**Comment by andrewbanchich** [comment]
I agree! That would be useful, along with some specifics on how to get up and running with contributing to the docs as well.

I'm planning on doing that and working on more doc updates soon.

**Comment by vinckr** [comment]
> Thank you! Merged! :) I think we should document a style guide for the docs somehow. I've never actually specified the things mentioned here (e.g. NYT style headlines) and the use is also not consistent across the repository. I was thinking we could put that into the core docs maybe? What's your take on this?

I started on a short style guide. Where in the docs should I put this?

**Comment by aeneasr** [comment]
Hm, maybe as a new document over at github.com/ory/x ? e.g. a document "Contributing"
